lib/views/includes/rich-header.erb

```diff
@@ -2,8 +2,8 @@
 <!-- BOOTSTRAP -->
 <script src='//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js'></script>
 
-<!-- MOMENT -->
-<script src='https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.js'></script>
```

Comment:
It was used all over for date conversion, but that's been replaced, and the chart display is the only place the requires it for the datepicker. It's still included explicitly there.